app/views/adoption_applications/new.html.erb

```diff
@@ -0,0 +1,120 @@
+<h1> Adoption Application </h1>
+
+<% if @adoption_application.errors.any? %>
+  <div id="error_explanation">
+    <h2><%= pluralize(@adoption_application.errors.count, "error") %> prevented your application from being submitted:</h2>
+ 
+    <ul>
+    <% @adoption_application.errors.full_messages.each do |message| %>
+      <li><%= message %></li>
+    <% end %>
```

Comment:
Thoughts on moving this into an errors partial? It might be useful so that you can separate it out from the `new` template.